test/commit_table_builder.go

```diff
@@ -62,13 +62,21 @@ func (builder *CommitTableBuilder) Add(commit Commit, location string) {
 }
 
 // branches provides the names of the all branches known to this CommitTableBuilder,
-// sorted alphabetically.
+// sorted alphabetically, with the main branch first.
 func (builder *CommitTableBuilder) branches() []string {
 	result := make([]string, 0, len(builder.commitsInBranch))
+	hasMain := false
 	for branch := range builder.commitsInBranch {
-		result = append(result, branch)
+		if branch == "main" {
```

Comment:
Was this the implementation we had in ruby with main hardcoded? Feel like we should lookup the main branch from config

Reply:
Please keep in mind that this is only for the test infrastructure where we don't need as much flexibility as in production code. So far, the tests [hardcode](https://github.com/Originate/git-town/blob/master/test/git_environment.go#L76) `main` as the main branch name. This seems the simplest possible implementation. I'd like to wait until we actually use another main branch name in tests before I add additional complexity and lookups.
 
This rewrite of the Cucumber infrastructure is a great opportunity to get rid of things that we thought we needed back then, but we didn't end up using them that much. I don't want to waste this opportunity by adding things back without a good reason to. YAGNI! 🙂